gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagManager.java

```diff
@@ -460,6 +463,10 @@ public synchronized void setActive(boolean active) {
         this.jobStatusPolledTimer = Optional.of(this.metricContext.timer(ServiceMetricNames.JOB_STATUS_POLLED_TIMER));
         ContextAwareGauge<Long> orchestrationDelayMetric = metricContext.newContextAwareGauge(ServiceMetricNames.FLOW_ORCHESTRATION_DELAY,
             () -> orchestrationDelay.get());
+        this.allSuccessfulMeter = metricContext.contextAwareMeter(
+            MetricRegistry.name(ServiceMetricNames.GOBBLIN_SERVICE_PREFIX, ServiceMetricNames.SUCCESSFUL_FLOW_METER));
+        this.allFailedMeter = metricContext.contextAwareMeter(
```

Comment:
What will happen with the metrics when the failover from master to slave occurs? Are we going to see a sudden spike in metrics that can trigger alerts?

Reply:
The meters would drop to 0, then return to normal after the length of the window time (a few minutes). It will be a slight blip in the graph, but I think it would not be enough to impact alerts as long as we do alerts on the right things (either looking for spikes in failures, or looking at ratio of success vs failure over a sufficiently large window).